pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerImpl.java

```diff
@@ -1223,13 +1223,13 @@ void sendComplete(final Exception e) {
                 if (finalEx != null && finalEx instanceof TimeoutException) {
                     TimeoutException te = (TimeoutException) e;
                     long sequenceId = te.getSequenceId();
-                    long ns = System.nanoTime();
+                    long ms = System.currentTimeMillis();
                     String errMsg = String.format(
-                        "%s : createdAt %s ns ago, firstSentAt %s ns ago, lastSentAt %s ns ago, retryCount %s",
+                        "%s : createdAt %s ms ago, firstSentAt %s ms ago, lastSentAt %s ms ago, retryCount %s",
```

Comment:
We should actually present this in seconds here. eg: `created 30.001 sec ago`

Reply:
+1 for seconds.

Reply:
Thanks @merlimat and @michaeljmarshall for the review.